platform.md

```diff
@@ -519,6 +519,7 @@ Usage:
 | `<gid>`              | `CNB_GROUP_ID`         |                          | Primary GID of the build image `User`                                                           |
 | `<group>`            | `CNB_GROUP_PATH`       | `<layers>/group.toml`    | Path to group definition (see [`group.toml`](#grouptoml-toml))                                  |
 | `<kaniko-cache-ttl>` | `CNB_KANIKO_CACHE_TTL` | 2 weeks                  | Kaniko cache TTL                                                                                |
+| `<kaniko-dir>`       |                        |                          | Kaniko directory (must be `/kaniko`)                                                            |
```

Comment:
If this must be /kaniko is there any point in exposing this?

Reply:
The intent is to signal to platform operators that they must mount in an additional volume

Reply:
Ah - it's not actually a flag (is absent from the list above the table). It's just an "input". So it's not configurable, but it's something you have to provide. I went hunting earlier for how other mounts (cache dir, platform dir, etc) are referenced in the platform spec and it's only via these inputs. So I guess this is just being consistent with how the other inputs are handled. But we should probably make this all more understandable somehow.

Reply:
Maybe also worth noting - the restorer has a `<kaniko-dir>` input already (https://github.com/buildpacks/spec/blob/main/platform.md#inputs-2). But we forgot to add it to the extender...

Reply:
ah! okay. Happy to approve in that case, but this seems like a redundant parameter.